test/kubemark/start-kubemark.sh

```diff
@@ -385,15 +412,15 @@ if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
   echo "DBG: set tenant partition flag false"
   export KUBERNETES_TENANT_PARTITION=false
   export KUBERNETES_RESOURCE_PARTITION=true
-  export KUBEMARK_CLUSTER_KUBECONFIG=${RP_KUBECONFIG}
 
-  create-kubemark-master
-  export KUBERNETES_RESOURCE_PARTITION=false
-  export KUBERNETES_SCALEOUT_PROXY=false
+  for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
```

Comment:
This enforce multiple kubemark master sequentially. Perhaps make it concurrent.

Reply:
good suggestion but it is not in the scope of this PR. tracked in issue #1036 https://github.com/CentaurusInfra/arktos/issues/1036